spec/commands_spec.rb

```diff
@@ -1,121 +1,128 @@
-require './spec/spec_helper'
+require 'spec_helper'
 
-describe "pry-rescue commands" do
-  describe "try-again" do
-    it "should throw try_again" do
-      PryRescue.should_receive(:in_exception_context?).and_return{ true }
+RSpec.describe "#Pry-rescue commands" do
+  describe "#try-again" do
+    it "expect to throw try_again" do
```

Comment:
Hehe.

It should be `it "is expected"` since expect commands more assertion than should in english. But when I find time will work on this and message you when it's ready.